### DISCUSSION TITLE
WandbLogger output.log is empty

### DISCUSSION THREAD

**Discussion by JinchaoLove** [Lightning-AI/pytorch-lightning#13808] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Hi, there, I'm using WandbLogger offline mode, and find the output.log is empty in `wandb/offline-run*/files/`. But when I turn the mode to online, wandb can successfully log the console outputs, and the output.log is correct both online the locally. May I ask why or how to deal with this problem?

**Comment by aniketmaurya** [comment]
answered here - https://github.com/wandb/wandb/issues/3978
